example/basic/main.go

```diff
@@ -44,7 +46,7 @@ var (
 		metric.WithDescription("A gauge set to 1.0"),
 	)
 
-	measureTwo = stats.NewMeasure("ex.com/two")
+	measureTwo = metric.NewFloat64Measure("ex.com/two")
 )
 
 func main() {
```

Comment:
I understand there is a `main.go` example, but it would help to use https://blog.golang.org/examples for each individual concept to make it clear which are the minimal parts for each thing it's trying to do.  With the main.go approach it's unclear which aspects are needed for which example it's trying to give. Like:

1. Minimal example of a counter
2. Minimal example of having a dimension on a metric
3. Minimal example of aggregating.

Maybe an example is

```
func ExampleGauge() {
oneMetric = metric.NewFloat64Gauge("ex.com/one",   metric.WithKeys(fooKey, barKey, lemonsKey))
gauge := oneMetric.Handle(ctx, meter, lemonsKey.Int(10))
gauge.Set(1)
}
```

Reply:
Add a `// Output: ` to the end of your example and it becomes code that is also executed when you run `go test`

Reply:
Another advantage of testable examples is that they usually integrate with an IDE and show up on help dialogs.

Reply:
This is a good suggestion, I always like the examples.